pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -301,58 +352,73 @@ ASTDDLCommand DDLCommand() :
 
 ASTSqlPlusCommand SqlPlusCommand() :
 {
- StringBuilder sb = new StringBuilder() ;
+  StringBuilder sb = new StringBuilder();
 }
 {
   (
   // e.g. SHOW ERRORS, GRANT EXECUTE ON ... TO ...
   // SQLPLUS commands
-  ( "@"
-  | <ACCEPT>
+  ( "@@" ( <_CHARACTER> | <SPECIAL_CHARACTERS> | <DELIMITER> ) *
+  | "@" ( <_CHARACTER> | <SPECIAL_CHARACTERS> | <DELIMITER> ) *
+  | <ACCEPT> | "ACC"
+  | "ARCHIVE LOG LIST"
```

Comment:
I'll have a closer look at the tokens here... e.g. if we write here `"ACC"`, then javacc creates a token for `ACC` and it won't be parsed as an identifier. Since this is not a reserved word, it is still valid to be used somewhere as an identifier. For that to work, we would need to add this token to the list in e.g. UnqualifiedID().